week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -33,6 +34,8 @@ console.log(y)
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
+The first output will be 10, because in the console.log on line 32 we call the function witch will execute a console.log.
+The second output will be undefined, because the variable y is "living" in the block scope of the function and we don't have access to her from outside the function.
```

Comment:
👍  Correct.

The variable `y` has been `defined` inside the function so it is not available outside of the function scope.